### PR TITLE
Run test in travis with OSX

### PR DESCRIPTION
**Pull request by unnonouno**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+  - linux
+  - osx
+
 python:
   - "2.7"
   - "3.4"
   - "3.5"
 
+before_install:
+  - "[ TRAVIS_OS_NAME = oxs ] && brew install hdf5 || true"
```
Comment (wkentaro) [inline .travis.yml]: `$TRAVIS_OS_NAME` and `osx`?

Comment (wkentaro) [inline .travis.yml]: maybe below is common style:
```
if [ "$TRAVIS_OS_NAME = "osx" ]; then brew install hdf5; fi
```

Comment (unnonouno) [inline .travis.yml]: Thanks. I didn't know Travis supports `if`.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**